cmd/kubeadm/app/util/apiclient/idempotency_test.go

```diff
@@ -81,3 +89,95 @@ func TestPatchNodeNonErrorCases(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateOrMutateConfigMap(t *testing.T) {
+	client := fake.NewSimpleClientset()
+	err := CreateOrMutateConfigMap(client, &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      configMapName,
+			Namespace: metav1.NamespaceSystem,
+		},
+		Data: map[string]string{
+			"key": "some-value",
+		},
+	}, func(cm *v1.ConfigMap) error {
+		t.Fatalf("mutate should not have been called, since the configmap should have been created instead of mutated")
```

Comment:
i would use "ConfigMap" in all instances instead of "configmap" or "config map" even if the existing code already uses the later syntax.
[1]